testing/e2e/scaffold/scaffold.go

```diff
@@ -172,18 +172,38 @@ func (s *Scaffold) FleetServerStatusIs(ctx context.Context, url string, state cl
 		s.Require().NoError(err)
 
 		return status.Status == state.String()
-	})
+	}, true)
+}
+
+// FleetServerStatusIs will poll fleet-server's status endpoint every second and return when it returns the expected state.
+// If the passed context terminates before a 200 is returned the current test will be marked as failed.
+func (s *Scaffold) FleetServerStatusNeverBecomes(ctx context.Context, url string, state client.UnitState) {
```

Comment:
Why is this named `FleetServerStatusNeverBecomes` when it tests that it becomes the passed state?